### PR TITLE
HW1 - Kris Nichols

### PR DESCRIPTION
**Pull request by nicholskl**
I had not realized that the description I wrote on the Git desktop app did not translate to my pull request but Chelsea just cleared this up. Now I will comment with my earlier comments "I had a lot of problems pushing the file but Chelsea was a lot of help. I also greatly appreciated the Markdown tutorial. After forking the file I saved the file and opened it in R studio. I then made my edits and pushed it to the repository with Git desktop." Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # hw01
 Repository for homework 1
+
+## Homework 1 9-27
+
+My name is Kristopher Nichols, I am 22 years old, and I love reading - my **favorite** author being Ernest Hemingway - and watching basketball. I am from Dallas, Texas and completed my undergraduate degree at [Stephen F. Austin State University](http://www.sfasu.edu/) with a major in psychology and a minor in French. I hope to apply to doctoral programs in behavioral science.
```
Comment (oddmentations) [inline README.md:6]: nice link!

**Comment by oddmentations** [comment]
Evaluation HW-01 for Kris Nichols
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      ✓     |              |            |
| **Coding strategy**         |      ✓     |              |            |
| **Presentation**            |       ✓    |              |            |
| **Achievement, creativity** |     ✓       |             |            |
| **Ease of access**          |      ✓     |              |            |

Remarks:

Given the minimal amount of code involved in this assignment, style is difficult to judge. Kris experimented with several aspects of the Markdown syntax, as required. He used four aspects of Markdown syntax: header, link, bold emphasis, and list. The brief bio gives me a good idea of his background and interests as a fellow student at U Chicago. The description of the challenges Kris encountered accompanied the Pull Request. I encountered similar challenges and sympathize with them. Great work!

**Comment by sushmitavgopalan16** [comment]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        x   |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |          |      x        |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Good job - solid submission, meets all submission criteria!
* Code is clean and clear.
* I gave you a 'Satisfactory' on Presentation because your page is a little plain and even just removing the comments from the assignment ("hw01" etc) would have made the page a little more aesthetically appealing.
* Glad you got the issues with the Pull Request sorted out. Make sure to commit early and often for future assignments!

**Review by harpergraf — COMMENTED** [review]
Evaluation HW-Harper Graf for Kris Nichols
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |  X         |              |            |
| **Coding strategy**         |   X        |              |            |
| **Presentation**            |           |     X         |            |
| **Achievement, creativity** |    X       |              |            |
| **Ease of access**          |    X       |              |            |

Remarks:

* You fulfilled the requirements and were very straightforward. The presentation could be enhanced with spacing or an image, but your description of yourself is good. 
* You are concise and straightforward in how you laid everything out. 
* I did not know you could highlight a passage as you have done. That's quite cool. Also, NBA or NCAA?
* I would have suggested not writing a paragraph since that does not explore the RMarkdown syntax as much as some other presentation styles might have. But you did meet 4/5 of the suggestions so that's not to say you haven't explored it. 
* In my own piece, I committed a lot (perhaps too much). But it was useful to me to see what I had done wrong and changed through commits in order to plan for future fixes. I would recommend committing more in the future on more complex assignments. Overall, great job!

**Pull request closed without merging**